### ISSUE TITLE
Create new flag when reference base doesn't match our reference fasta

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#517]
At the moment this will be flagged as a major HGVS change but it would be better to have finer granularity for the issue 

While we're doing this - should normalization be a separate flag or part of this? Both could change the reference though normalization could also change the coordinate and alt.

* New flag type
* Need a new logo maybe something like "!ref"

Something to consider is that maybe in the future we'll have the different patch versions and then handle that, so maybe instead of "our reference fasta" we should store what version we checked, can get this from genome build config I think

Related to #68 - warn when variant ref doesn't match our reference fasta - though that is about variants - this is about classifications

**Comment by davmlaw** [comment]
Maybe it shouldn't be a flag - but maybe we should attach something to the variant, or give a warning on variant details page at the very least

I am not sure how these come now since the bcftools normalization switch

**Comment by davmlaw** [comment]
Flags are ok for generic, but a bit slow for all variants - use specific model for faster implementation.

We'll create these at the moment through management command, flag on variant page - then we can quickly report on how often this affects classification, then work out what to do from there (raise new issue upon discussion with group)

Original issue was to do with symbolic variants, which are a tiny % so perhaps have tool be able to just check those

**Comment by davmlaw** [comment]
@TheMadBug - removed from Jun 2024 release, complain if you care

**Comment by EmmaTudini** [comment]
@davmlaw James is away today/tomorrow. I'm ok with leaving this, but would like to make sure we know how many variants in prod have the incorrect reference base

**Comment by davmlaw** [comment]
Would you like to know by the next deploy?

**Comment by EmmaTudini** [comment]
I don't think we need this - and I don't actually think any symbolic variants exist in Shariant prod at the moment anyway. Could we do it in the next round of testing i.e. when we switch over to Biocommons?
